@xen-orchestra/backups/_PoolMetadataBackup.js

```diff
@@ -73,3 +71,6 @@ export class PoolMetadataBackup {
     )
   }
 }
+
+exports.PATH_DB_DUMP = PATH_DB_DUMP
```

Comment:
I think you should export a value as close as possible to it's definition.

Reply:
For what reason?

Reply:
Better code locality, closer to ES modules.